src/Raven.Server/Documents/TcpHandlers/SubscriptionDocumentsFetcher.cs

```diff
@@ -144,7 +144,8 @@ public class SubscriptionDocumentsFetcher
 
                     if (ShouldSendDocumentWithRevisions(_subscription, run, _patch, docsContext, item, revisionTuple, out var transformResult, out var exception) == false)
                     {
-                        includesCmd.AddRange(run.Includes);
+                        if(run != null)
```

Comment:
@karmeli87 did you encounter a situation when run was null?

Reply:
I did, yes. This can happen if there is no script to run `from Users`

Reply:
e.g. `from Users (Revisions = true) as doc`